### ISSUE TITLE
Update Doks documentation

### ISSUE THREAD

**Issue by h-enk** [thuliteio/getdoks.org#341]
Bring the current documentation up to par with the current code base.

**Comment by h-enk** [comment]
TODO

- [x] Add DocSearch
- [x] Add Plausible tracking script

**Comment by hawkerm** [comment]
Looks like a few pages are still blank like the scripts one https://github.com/gethyas/getdoks.org/blob/main/content//docs/guides/scripts.md

Is there anywhere else to look for information about extending doks capabilities like custom/overriding rendering of components?

Thanks!

**Comment by h-enk** [comment]
> Is there anywhere else to look for information about extending doks capabilities like custom/overriding rendering of components?

No, unfortunately not. I will though make it my priority to bring the docs to a workable level — ASAP

**Comment by hawkerm** [comment]
Thanks for the prompt response. I'm still new to hugo too, are the scripting hooks unique to doks, or does hugo have some base functionality I should look at as well? I believe doks still uses hugo under the hood correct?

**Comment by h-enk** [comment]
https://github.com/thuliteio/getdoks.org/pull/364
